common/common.go

```diff
@@ -0,0 +1,53 @@
+package common
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// GetDefaultMicroProfileStartupProbe returns the default values for MicroProfile Health-based startup probe.
+func GetDefaultMicroProfileStartupProbe(ba BaseComponent) *corev1.Probe {
+	return &corev1.Probe{
+		Handler: corev1.Handler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Path:   "/health/started",
+				Port:   intstr.FromInt(int(ba.GetService().GetPort())),
+				Scheme: "HTTPS",
+			},
+		},
+	}
+}
+
+// GetDefaultMicroProfileReadinessProbe returns the default values for MicroProfile Health-based readiness probe.
+func GetDefaultMicroProfileReadinessProbe(ba BaseComponent) *corev1.Probe {
+	return &corev1.Probe{
+		Handler: corev1.Handler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Path:   "/health/ready",
+				Port:   intstr.FromInt(int(ba.GetService().GetPort())),
+				Scheme: "HTTPS",
+			},
+		},
+		InitialDelaySeconds: 30,
+		PeriodSeconds:       10,
+		TimeoutSeconds:      3,
+		FailureThreshold:    1,
```

Comment:
@hibell we should update the FailureThreshold of readiness probe as well to be 3. Sorry I wasn't clear.